src/hyp3_isce2/dem.py

```diff
@@ -58,16 +59,39 @@ def buffer_extent(extent: list, buffer: float) -> list:
     ]
 
 
+def utm_from_lon_lat(lon: float, lat: float) -> int:
+    hemisphere = 32600 if lat >= 0 else 32700
+    zone = int(lon // 6 + 30) % 60 + 1
+    return hemisphere + zone
+
+
+def get_dem_resolution(extent, res):
+    # coordinates of the upper left corner pixel
+    lonc = extent[0]
+    latc = extent[3]
+
+    epsg_code = utm_from_lon_lat(lonc, latc)
+    myprj1 = pyproj.Transformer.from_crs(4326, epsg_code, always_xy=True)
+    myprj2 = pyproj.Transformer.from_crs(epsg_code, 4326, always_xy=True)
+    xc, yc = myprj1.transform(lonc, latc)
+    x2 = xc + res
+    y2 = yc - res
+    lon2, lat2 = myprj2.transform(x2, y2)
+    return abs(lon2 - lonc), abs(lat2 - latc)
```

Comment:
Since we aren't using the latitude resolution anymore, it might be good to remove it from the return statement.

Reply:
good idea.